### DISCUSSION TITLE
My project is not running. only stuck to login page.

### DISCUSSION THREAD

**Discussion by TejpallPathak** [fullstackhero/blazor-wasm-boilerplate#129] [General]
Hi Team,

My project is not running. only stuck to login page.

running API project already. after submit on login page in wasm project. it is getting tokens but still can not redirect to any other page.

also in console this error is coming just a FYI.
![2022-02-15](https://user-images.githubusercontent.com/3037658/153962757-e22b5aa9-0ba8-42b2-a2ed-e28a0d07443d.png)

**Comment by bharathwajv** [comment]
Are you running latest version of API project?

**Comment by pyramidsbuilder** [comment]
Same issue here, Yes on the latest API project, I tracked it down it seems there is a problem with Blazor reading the response data while the project returns ActionResult: OK(token), Blazor client_.SendAsync  (FSHAPi.cs: line 1695) cannot read the object/value/token
